src/routes.php

```diff
@@ -13,4 +13,5 @@
     'items/show' => ['ItemController', 'show', ['id']],
     'items/add' => ['ItemController', 'add',],
     'items/delete' => ['ItemController', 'delete',],
+    'about_us' => ['AuthorController','show'],
```

Comment:
if only one page for about-us, please change show method into index (+ you already have an index file in your View/aboutUS/ folder